src/css/components/_captions-settings.scss

```diff
@@ -4,33 +4,49 @@
   color: $primary-foreground-color;
   height: 100%;
 }
-
 // Layout divs
-.vjs-text-track-settings .vjs-modal-dialog-content {
-  display: grid;
-  grid-template-columns: minmax(230px, 400px) 230px 400px;
-  grid-template-rows: minmax(180px, 330px) 180px 330px;
+@supports (display: grid) {
+  // code that will only run if CSS Grid is supported by the browser
+  .vjs-text-track-settings .vjs-modal-dialog-content {
+    display: grid;
+    grid-template-columns: minmax(230px, 400px) 230px 400px;
+    grid-template-rows: minmax(180px, 330px) 180px 330px;
+  }
 
-}
+  .vjs-text-track-settings .vjs-track-settings-colors {
+    display: block;
+    grid-column: 1;
+    grid-row: 1;
+  }
 
-.vjs-text-track-settings .vjs-track-settings-colors
- {
-  display: block;
-  grid-column: 1;
-  grid-row: 1;
-}
+  .vjs-text-track-settings .vjs-track-settings-font {
+    grid-column: 2;
+    grid-row: 1;
+  }
 
-.vjs-text-track-settings .vjs-track-settings-font
-{
-  grid-column: 2;
-  grid-row: 1;
-}
-.vjs-text-track-settings .vjs-track-settings-controls {
-  grid-column: 2;
-  grid-row: 2;
+  .vjs-text-track-settings .vjs-track-settings-controls {
+    grid-column: 2;
+    grid-row: 2;
+  }
 }
+@supports not (display: grid) {
```

Comment:
`@supports` doesn't work on the browsers that we want to target that doesn't have the grid. We need to keep the regular CSS without any check and then reset it and make it a grid inside the `@supports` block. The `@supports` block will be completely ignored on older browsers that don't support `@supports`.

Reply:
Video.js needs to support or at least not break all the way back to IE8!

Reply:
So have the current table CSS then below it the `@supports` so if it does support grid it will override?

Reply:
yup, more or less.